services/movie_session.py

```diff
@@ -42,3 +46,16 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(
+        movie_session_id: int
+) -> list:
+    tickets = Ticket.objects.filter(
+        movie_session_id=movie_session_id
+    )
+    taken_seats = [
+        {"row": info.row, "seat": info.seat}
+        for info in tickets
+    ]
+    return taken_seats
```

Comment:
info is a bad name for variable, maybe it is better to use ticket? 
also u can use values method to simplify ur code